frontend/types/campaignCharities.ts

```diff
@@ -11,10 +11,10 @@ export type CampaignCharityBaseData = {
 };
 
 export type CampaignCharityData = CampaignCharityBaseData & {
-  charity: CharityListData;
+  charity: CharityData;
 };
 
-export type CampaignCharityDonationPublicData = Omit<CampaignCharityData, 'givingSgUrl'> & DonationBreakdownData;
+export type CampaignCharityDonationPublicData = CampaignCharityData & DonationBreakdownData;
```

Comment:
I thought we were going to link to the website? Only the redeem page takes the people to the givingsg campaign? But I don't think there's a prob with linking to the givingsg campaign also.